web/src/services/i18n/translations/ja.yml

```diff
@@ -156,19 +156,19 @@ font weight: フォントウェイト
 font color: フォントカラー
 italic: イタリック
 New node: 新しいノード
-Style: スタイル
-Empty: 空
-Default: デフォルト
-Professional: プロフェッショナル
-Basic Geometry: 基本ジオメトリ
-Points: ポイント
-Point with label: ラベル付きポイント
-Polyline: ポリライン
-Polygon: ポリゴン
-Extruded polygon: 押し出しポリゴン
-Simple Style: シンプルスタイル
-Plateau: 台地
-Color buildings by height: 高さで建物の色を設定
+Style: ''
+Empty: ''
+Default: ''
+Professional: ''
+Basic Geometry: ''
+Points: ''
+Point with label: ''
+Polyline: ''
+Polygon: ''
+Extruded polygon: ''
+Simple Style: ''
+Plateau: ''
+Color buildings by height: ''
```

Comment:
Could you remove `t` from the preset styles? We should not implement by remove the translation text since it could be use in other places.